app/settings/surveys/targeted-surveys/targeted-edit.controller.js

```diff
@@ -209,28 +209,12 @@ function (
         if ($scope.survey.attributes !== undefined && $scope.survey.attributes.length) {
             $scope.stepThreeWarning = false;
             $scope.activeStep = 4;
-            calculateStats();
+            $scope.sms = $scope.survey.attributes.length * $scope.finalNumbers.goodNumbers.length;
```

Comment:
could we call this something more total total_sms_messages?

Reply:
yeah, good point...